camp/settings/base.py

```diff
@@ -237,6 +237,8 @@
 
 # huey
 
+MAX_QUEUE_SIZE = int(os.environ.get('MAX_QUEUE_SIZE', 300))
```

Comment:
We currently have 224 PurpleAir monitors in the production system, which would mean 448 individual entries to process every two minutes. Would it make sense to bump this up to roughly how many tasks we need to process within that time frame, e.g., 500, by default?

Reply:
Done.